security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md

```diff
@@ -611,7 +611,7 @@ _图四_
 
 ## 角色和用例
 
-重点是安全、保护、检测和尽可能的自动响应。它不一定是独立的开发工具，而是透明地集成到开发流程中的安全工具，使得开发过程中就能够执行安全策略、快速反馈和直接补救。有关云原生安全用例的具体信息，请参考 [SIG-Security 的用例列表](https://github.com/cncf/sig-security/blob/master/usecases.md)。
+重点是安全、保护、检测和尽可能的自动响应。它不一定是独立的开发工具，而是透明地集成到开发流程中的安全工具，使得开发过程中就能够执行安全策略、快速反馈和直接补救。有关云原生安全用例的具体信息，请参考 [SIG-Security 的用例列表](https://github.com/cncf/tag-security/blob/main/usecases.md)。
```

Comment:
This is a 404. But It also was a 404 before the change.

Reply:
this link should be https://github.com/cncf/tag-security/tree/main/usecase-personas

Reply:
see issue https://github.com/cncf/tag-security/issues/649